app/src/main/res/values/strings.xml

```diff
@@ -93,6 +93,8 @@
         for story templates does not contain usable templates.  Select the appropriate
         \'&lt;font color=\"green\">&lt;b>SP Templates&lt;/b>&lt;/font>\' folder - normally found on the
         &lt;font color=\"green\">&lt;b>SD card&lt;/b>&lt;/font>.</string>
+    <string name="stories_not_found_in_progress">There are no stories in progress.</string>
+    <string name="stories_not_found_completed">There are no completed stories.</string>
```

Comment:
@bryanwussow If there is a change you want to make to this verbiage just let me know!

Reply:
These look good.